docs/releases/README.md

```diff
@@ -1,12 +1,12 @@
 # Building & Versioning
 
-- [Building & Versioning](#building--versioning)
+- [Building \& Versioning](#building--versioning)
```

Comment:
Nice catch!

Reply:
Appreciate it but it wasn't me. The extension automatically did it for me on save: https://marketplace.visualstudio.com/items?itemName=huntertran.auto-markdown-toc